definitions/classnames_v2.x.x/flow_>=v0.23.x/classnames_v2.x.x.js

```diff
@@ -1,5 +1,8 @@
+
+type Classes = string | [class: string]: bool } | Array<string>
```

Comment:
Can we namespace this so it doesn't conflict as a global? Generally I've been recommending namespacing like `$npm$classnames$Classes`